hefquin-engine/src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/loptimizer/heuristics/MergeRequests.java

```diff
@@ -179,12 +182,25 @@ else if ( rootOp instanceof LogicalOpFilter filterOp )
 		else if ( rootOp instanceof LogicalOpBind bindOp )
 		{
 			// The BIND clause can be merged into a request operator if that request is for a SPARQL endpoint.
-			final LogicalOperator childOp = rewrittenSubPlans.get(0).getRootOperator();
+			final LogicalOperator childOp = rewrittenSubPlans.get( 0 ).getRootOperator();
```

Comment:
In the future, please try to avoid such types of changes that are not actually related to the purpose of the PR, because that makes it harder for me to see where the actual changes are.

By the way, for code like this, I would not actually add spaces. My rule of thumb is to add spaces only for cases in which the function arguments are something composite, i.e., not a single constant or a single program variable but other function calls or operations. For instance:
```java
get( list.size() );  // with spaces
get( i - 1 );  // with spaces
get(i); // no spaces
```

Reply:
> In the future, please try to avoid such types of changes that are not actually related to the purpose of the PR, because that makes it harder for me to see where the actual changes are.

Noted

Reply:
> By the way, for code like this, I would not actually add spaces. My rule of thumb is to add spaces only for cases in which the function arguments are something composite, i.e., not a single constant or a single program variable but other function calls or operations

Okay, I'll keep that in mind.